### PR TITLE
Fix formatting issues and add missing info

### PR DESCRIPTION
**Pull request by bszwarc**
**Description**

Changes proposed in this pull request:

- fix formatting issues
- add missing information

See also #328

### CODE REVIEW

File: docs/general-settings.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ settings: {
...
-**backdropDisabled** When set to `true`, prevents the backdrop layer from covering the top and left navigation when showing modal windows. By default, the backdrop is enabled and covers the top and left navigation when showing modal windows. To keep the default configuration, set the value to `false` or leave this option unconfigured.
-**header.logo** defines the top left navigation logo. It has a fixed height of 28px.
-**header.title** defines the top left navigation title.
-**header.favicon** defines the favicon. Requires a standard favicon file with the `.ico` extension, and 16x16px or 32x32px dimensions.
+* **hideNavigation** disables Luigi's default out-of-the-box navigation when set to `true`. As a result, neither the left nor top navigation are visible in your application, and you can implement your own navigation UI. By default, the navigation is enabled.
+* **backdropDisabled** When set to `true`, it prevents the backdrop layer from covering the top and left navigation when showing modal windows. By default, the backdrop is enabled.
```
Comment (klaudiagrz) [inline docs/general-settings.md]: Something is needed here between the parameter's name and When. Paraphrasing should do the thing.

Comment (bszwarc) [reply docs/general-settings.md]: 👍

File: docs/general-settings.md [hunk 2]
```diff
@@ -13,8 +13,8 @@ settings: {
...
-**backdropDisabled** When set to `true`, prevents the backdrop layer from covering the top and left navigation when showing modal windows. By default, the backdrop is enabled and covers the top and left navigation when showing modal windows. To keep the default configuration, set the value to `false` or leave this option unconfigured.
-**header.logo** defines the top left navigation logo. It has a fixed height of 28px.
-**header.title** defines the top left navigation title.
-**header.favicon** defines the favicon. Requires a standard favicon file with the `.ico` extension, and 16x16px or 32x32px dimensions.
+* **hideNavigation** disables Luigi's default out-of-the-box navigation when set to `true`. As a result, neither the left nor top navigation are visible in your application, and you can implement your own navigation UI. By default, the parameter is set to `false`, which means the navigation is enabled.
```
[the remark this answers is not in the document]
Comment (bszwarc) [reply docs/general-settings.md]: Done

File: docs/general-settings.md [hunk 3]
```diff
@@ -13,8 +13,8 @@ settings: {
...
+* **backdropDisabled** prevents the backdrop layer from covering the top and left navigation when showing modal windows. By default, the backdrop is set to `true`.
+* **header.logo** defines the top left navigation logo. It has a fixed height of 28px.
+* **header.title** defines the top left navigation title.
+* **header.favicon** defines the favicon. Requires a standard favicon file with the `.ico` extension, and 16x16px or 32x32px dimensions.
```
[the remark this answers is not in the document]
Comment (bszwarc) [reply docs/general-settings.md]: Done

**Pull request merged**